tokio/src/runtime/builder.rs

```diff
@@ -152,6 +157,14 @@ impl Builder {
         self
     }
 
+    ///Specifies limit on number of threads, used for blocking annotation.
+    ///
+    ///The default value is 512.
+    pub fn blocking_threads(&mut self, val: usize) -> &mut Self {
```

Comment:
I'm thinking we should name this `max_threads` and the other config: `core_threads`. What do you think?

Reply:
Do you mean to use as `blocking_threads = max_threads - core_threads`?

Sounds fine, though I'd prefer something more explicit `core_threads` -> `io_threads`